frontend/public/components/cluster-settings/basicauth-idp-form.tsx

```diff
@@ -17,7 +18,10 @@ export const DroppableFileInput = (props: any) => (
   />
 );
 
-export class AddBasicAuthPage extends PromiseComponent<{}, AddBasicAuthPageState> {
+class AddBasicAuthPageWithTranslation extends PromiseComponent<
+  AddBasicAuthPageProps,
```

Comment:
Does `WithTranslation` work here then you don't need to define your own types?

```tsx
import { withTranslation, WithTranslation } from 'react-i18next';
```
```suggestion
  WithTranslation,
```

Reply:
<img width="759" alt="Screen Shot 2020-10-13 at 5 14 08 PM" src="https://user-images.githubusercontent.com/895728/95916995-8c3cdb80-0d77-11eb-9f5a-4bc7870b3798.png">